desktop/core/src/desktop/api2.py

```diff
@@ -813,6 +814,44 @@ def export_documents(request):
     return make_response(f.getvalue(), 'json', filename)
 
 
+def topologicalSort(docs):
+
```

Comment:
1. put a type check for the docs variable.
2. put an emptiness check for the docs variable

Reply:
as we are sending the docs which is list (docs = []) from import_documents function then why need to check the docs type?
And if it is empty then it is automatically handled by code it self.